Tasks/AppStoreRelease/Tests/L0.ts

```diff
@@ -20,20 +20,20 @@ describe('app-store-release L0 Suite', function () {
     });
     /* tslint:enable:no-empty */
 
```

Comment:
Probably we need to add timeout somewhere here since `this` doesn't work inside arrow functions and CI is failing quite often on `gulp testwithresults`

Reply:
Done, thanks!